conn-fix/README.md

```diff
@@ -0,0 +1,359 @@
+# FIX Connect
+
+This "Connect" component extension supports connection to the target system via [FIX protocol](https://www.fixtrading.org/what-is-fix/).
```

Comment:
This "Connect" component extension supports **connections** to the target system via [FIX protocol](https://www.fixtrading.org/what-is-fix/).